corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -1072,7 +1077,8 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
                             return {
                                 field: row.textField.val(),
                                 type: row.type(),
-                                direction: row.direction()
+                                direction: row.direction(),
+                                display: row.display()
```

Comment:
I think to make this support multiple languages you can combine it with `screen.lang`